doc/node.1

```diff
@@ -120,6 +120,10 @@ Throw errors for deprecations.
 Silence all process warnings (including deprecations).
 
 .TP
+.BR \-\-napi\-modules
+Load N-API modules (experimental, opt-in by adding this flag).
```

Comment:
Fixed in https://github.com/nodejs/node/pull/11975/commits/e1ca374cb5be63f6f5167bd4e6b40a01f9ecb878